app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  belongs_to :post
+
```

Comment:
It would be nice to use the `after_save` call back to call the `#update_likes_counter` method every time a new post has been saved to the database. Please read more about database callbacks [here](https://guides.rubyonrails.org/active_record_callbacks.html).